portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,7 +63,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     for (Entity entity : results.asIterable()) {  
         String name = (String) entity.getProperty("user_name");
         String comment = (String) entity.getProperty("user_comment");
-        messages.add("\""+ comment + "\" \n - " + name);
+        String image = (String) entity.getProperty("user_img");
+        messages.add("\""+ comment + "\" \n - " + name + " img: " +image);
```

Comment:
If you are planning to show the image to the user, it would be helpful if you get the URL in a separate field on the client side, rather than as part of a long string. In order to this, we could create a class called comment and three fields: commentText, name and image to it. Then on the client(javascript), we would be able to access comment.commentText, comment.name and comment.image separately. 

We can chat about this if you like.